.github/workflows/templates-api.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 env:
   MINIKUBE_PROFILE: templates-api
+  KUBERNETES_NAMESPACE: templates-api
```

Comment:
Namespace je tu prilis casto a chcem to skopirovat pre buffer api, tak som to dal do env.